CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+## Forum
+
+* For questions related to Mastodon development, please use
+  the [zulip chat](https://imagesc.zulipchat.com/#narrow/stream/327470-Mastodon).
+
+## License Agreement
+
+All contributions like will fall under the "BSD 2-Clause (Simplified) License" as described in
+the [LICENSE](LICENSE.txt) file.
+
+## Code formatting
+
+We use the Eclipse code formatter. You can find the configuration
+file [here](https://github.com/scijava/scijava-coding-style/blob/master/src/main/resources/eclipse-formatter-settings/mastodon-coding-style.xml)
+. Please use this formatter for your pull requests.
+
+Users of IntelliJ IDEA can use
+the [Eclipse Code Formatter Plugin](https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter) to automatically
+format your code.
+
+We recommend to use Auto-Format on Save in your IDE, preferably only on changed lines.
```

Comment:
This is really good! And useful!
What do you think of actually putting this in the org page, and say that it applies to *all* mastodon repos?

Reply:
Good suggestion.
I removed the code from here and made a pull-request to the org page.
@tinevez Please have a look at: https://github.com/mastodon-sc/.github/pull/1